cms/bundles/forms.py

```diff
@@ -69,6 +69,8 @@ def _validate_bundled_pages(self) -> None:
                 page = page.specific
                 if page.in_active_bundle and page.active_bundle != self.instance:
                     raise ValidationError(f"'{page}' is already in an active bundle ({page.active_bundle})")
+                if self.cleaned_data.get("release_calendar_page") == page:
```

Comment:
As I was going to approve this, I realised we're not covering one other scenario:

what if the RC page is linked in another active bundle (i.e. as the `release_calendar_page` FK?